index.js

```diff
@@ -46,3 +49,9 @@ module.exports = function percySnapshot(b, name, options) {
     }
   });
 };
+
+// jasmine cannot mock individual functions, hence adding isPercyEnabled to the exports object
+// also need to define this at the end of the file or else default exports will over-ride this
+module.exports.isPercyEnabled = async function isPercyEnabled(driver) {
```

Comment:
we can't stub `utils.isPercyEnabled`?

Reply:
Already tried getting error
[the relevant FAQ entry](https://jasmine.github.io/pages/faq.html#module-spy)
![Screenshot 2023-09-21 at 5 19 18 PM](https://github.com/percy/percy-webdriverio/assets/92926953/1341d4bc-4ec5-43ee-a2cf-03515f30f0b8)

Reply:
suggested new changes below, you in tests in which case you can revert this changes.

Reply:
Continuing communication in below thread